refresh/index.js

```diff
@@ -110,19 +110,18 @@ module.exports = generators.Base.extend({
       // Service configuration
       this.services = this.spec.services || {};
 
-      // Monitoring
-      this.metrics = (this.spec.metrics === true);
+      // Capability configuration
+      this.capabilities = this.spec.capabilities || {};
 
       // Autoscaling
-      if (this.spec.autoscale === true) {
-        this.autoscale = `${this.projectName}ScalingService`;
-      } else if (typeof(this.spec.autoscale) === 'string') {
-        this.autoscale = this.spec.autoscale;
+      if (this.capabilities.autoscale === true) {
+        this.capabilities.autoscale = `${this.projectName}ScalingService`;
       }
+
       // Autoscaling implies monitoring and Bluemix
-      if (this.autoscale) {
+      if (this.capabilities.autoscale) {
         this.bluemix = true;
-        this.metrics = true;
+        this.capabilities.metrics = true;
       }
```

Comment:
`autoscaling` covered in PR https://github.com/IBM-Swift/generator-swiftserver/pull/87
Further changes required to reinstate checking/robustness for `metrics`.